src/ui/units/collections/components/CollectionContent/CollectionContent.tsx

```diff
@@ -235,7 +234,7 @@ export const CollectionContent = React.memo<Props>(
                                     description: item?.description ?? '',
                                     onApply: (collection: UpdateCollectionResponse | null) => {
                                         if (collection) {
-                                            dispatch(updateCollectionInItems(collection));
+                                            refreshContent();
```

Comment:
After renaming a collection or workbook in the list, we make two unnecessary requests:
<img width="1435" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/a2a35b9d-d20d-4c25-b9b0-6a773b865f58">

Reply:
agree, fixed